client/my-sites/site-settings/form-general.jsx

```diff
@@ -358,7 +358,12 @@ export class SiteSettingsFormGeneral extends Component {
 						disabled={ isRequestingSettings }
 						onClick={ eventTracker( 'Clicked Site Visibility Radio Button' ) }
 					/>
-					<span>{ translate( 'Do not allow search engines to index my site' ) }</span>
+					<span>{ translate( 'Discourage search engines from indexing my site' ) }</span>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **121** times:
`translate( 'Discourage search engines from indexing this site' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/blog-public-wording)

Reply:
Pushed [update/blog-public-wording 4dccc58be9] Small wording change to take advantage of pre-translated strings